app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+from typing import Union
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
```

Comment:
Refer to `Animal.alive` instead of `self.alive`, because `alive` is a class attribute.

Reply:
not fixed